pinot-query-planner/src/main/java/org/apache/pinot/calcite/sql/fun/PinotOperatorTable.java

```diff
@@ -97,6 +97,13 @@ public static PinotOperatorTable instance() {
           InferTypes.FIRST_KNOWN,
           OperandTypes.MINUS_OPERATOR.or(OperandTypes.family(SqlTypeFamily.TIMESTAMP, SqlTypeFamily.TIMESTAMP)));
 
+  // These 2 operators are not directly registered in the operator table, but are used in the custom optimization rules
+  // (see PinotSemiJoinToInClauseRule and PinotLeftJoinToNotInClauseRule) to work around the Calcite limitation of not
+  // supporting IN and NOT IN as the function call.
+  public static final PinotSqlFunction PINOT_IN = new PinotSqlFunction("IN", ReturnTypes.BOOLEAN_NULLABLE, null);
+  public static final PinotSqlFunction PINOT_NOT_IN =
+      new PinotSqlFunction("NOT_IN", ReturnTypes.BOOLEAN_NULLABLE, null);
```

Comment:
Do we really need these? What happens if we use `SqlStdOperatorTable.IN` / `SqlStdOperatorTable.NOT_IN` in those two rules directly instead?